docs/dapp_overview.md

```diff
@@ -22,54 +23,68 @@ The unique characteristics of a Node are defined by three structures set by the
 - **Replicas**: number of Node's replicas that will be created in the **k8s cluster**.
 - **Environment**: user-defined environment variables that can be accessed from within the Node.  
 
-**Nodes are created insite a Kubernetes cluster as Deployments.**  
+**Nodes are created inside a Kubernetes cluster as Deployments.**  
 As described previously in "What are dApps?", a dApp that is a Node can't have child dApps. This means that:
 1) A Node is a dApp that has a Node structure defined in it.
-    - This implies that Nodes make use of all the structures defined within it's parent (Channels, Types, Boundaries and Aliases)
+    - This implies that Nodes make use of all the structures defined within it's parent (Channels, Routes, Types, Boundaries and Aliases)
 2) In the tree-like organization, Nodes are always leaves.
 
 
 ## What are Channels?  
-Channels are the structures that allow dApps and Nodes to exchange information between each other.  
+Channels are the structures that allow dApps and Nodes to exchange information between each other via message brokers.  
 A dApp/Node inputs and outputs are defined by it's Boundary. The Boundary, on the other hand, references a parent dApp's Channel (or Alias, which will be described ahead).
 
 ![channel](img/chan-func.png)  
 
-Channels are created to receive and send a specific kind of information. This specification is defined by a **Channel's Type**, which is a reference to a Type structure defined in the same context (in the same dApp) as the Channel. In other words:
-- A Channel only carries a specific kind of data (such as *int*, *string*, or an object defined by the user).
-- A Channel can't exist without a Type.  
+Channels are created to receive and send a specific kind of information. This specification is defined by it's **Type**, which is a reference to a Type structure defined in the same context (in the same dApp) as the Channel. In other words:
```

Comment:
its `type`, no?

Reply:
Didn't really understand your comment :confused:

Reply:
Yes, it is!